<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lds_calendarwidget</class>
 <widget class="QDialog" name="lds_calendarwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#lds_calendarwidget{
border:2px solid #ff650b;
}</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>275</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame_2{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
stop:0 #fd8238, 
stop:0.5 #fd8238, 
stop:0.51 #ff650a
stop:1 #ff650a
);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_hour">
      <property name="text">
       <string>时</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_second">
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_month">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_minute">
      <property name="text">
       <string>分</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_year">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_day">
      <property name="text">
       <string>日</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="lds_lineedit" name="lineEdit_year">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="lds_lineedit" name="lineEdit_month">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="lds_lineedit" name="lineEdit_day">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="lds_lineedit" name="lineEdit_second">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="lds_lineedit" name="lineEdit_minute">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="lds_lineedit" name="lineEdit_hour">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>85</y>
     <width>300</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_today">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>今天</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curyear_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2015</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curyear">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2016</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curyear_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2014</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>65</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="outer_stylesheet" stdset="0">
     <string>pushbutton_key</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_lineedit</class>
   <extends>QLineEdit</extends>
   <header>lds_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
